Gamma mumu

mrsolt opened the pull request:
This pull request makes it possible to bias gamma -> mu+ mu- conversion in the target. It is based on the other biasing operators currently in SimCore. It is mostly independent from the rest of SimCore.

Files added:
include/SimCore/BiasOperators/GammaToMuPair.h
src/SimCore/BiasOperators/GammaToMuPair.cxx 

Files modified:
python/bias_operators.py.in

mrsolt committed:
add gamma to mu pair bias operators

mrsolt committed:
update gamma to mu pair bias operator

mrsolt committed:
clean up debugs

tomeichlersmith reviewed (approved):
Please update the documentation. There are a few parts that are still leftover from the copy of the PN operator.

Otherwise, this is a simple addition that won't break anything else, so I say we gopher it ( :wink:   little UMN joke there). I would like @mrsolt to add a test configuration in the `Biasing/test` directory (similar to the other test configs there). This would give a helpful example to those trying to use this operator and will make it easier to fold into testing when we get that working.

mrsolt committed:
update documentation

mrsolt commented:
I think I updated all the documentation now.

I also added a test in the Biasing submodule and submitted a separate pull request. Hopefully, I did this correctly since I don't really know how to check it.

omar-moreno commented on include/SimCore/BiasOperators/GammaToMuPair.h:
```diff
@@ -0,0 +1,69 @@
+  GammaToMuPair(std::string name, const framework::config::Parameters& p);
+
+  /** Destructor */
+  ~GammaToMuPair() {}
```
Use `~GammaToMuPair() = default`

mrsolt replied:
Ok, I have changed it to default.

omar-moreno reviewed (changes requested):
One small tweak and the code looks fine. 

It would be useful to link a talk with comparisons to old studies for future reference.

mrsolt committed:
changing to default destructor

omar-moreno commented:
Given that this is basically stand alone and that @mrsolt is actively working on studying the kinematics of these events, I don't see an issue with merging this in.
